doc/manual/chapters/translation.tex

```diff
@@ -1,10 +1,364 @@
+	Compiled as
+	\begin{lstlisting}[language=golang]
+if !condition {
+	panic("condition");
+}
+	\end{lstlisting} \\
+	\hline
+	Await statement & \lstinline[language=pcal]|await condition;| &
+	Compiled as
+	\begin{lstlisting}[language=golang]
+awaitLabel:
+	if !condition {
+		goto awaitLabel
+	}
+	\end{lstlisting} \\
+	\hline
+	Goto statement & \lstinline[language=pcal]|goto label;| & Supported; compiled as expected \\
+	\hline
+	Single process algorithm &
+	\begin{lstlisting}[language=pcal]
+--algorithm Algo {
+  variables x = exp1, y \in exp2;
+  {
+  	body;
+  }
+}
+	\end{lstlisting} &
+	Supported; compiled as a single-threaded single-process Go program \\
+	\hline
+	Multiprocess algorithm &
+	\begin{lstlisting}[language=pcal]
+--algorithm Algo {
+  variables x = exp1, y = exp2;
+  process (P \in exp3)
+  variables local = exp4;
+  {
+  	body;
+  }
+}
+	\end{lstlisting} &
+	Supported; compiled with various strategies configured by the user \\
+	\hline
+	\caption{PlusCal constructs}
+	\label{table:pcal-constructs}
+\end{longtable}
+\end{center}
+
+Below are the TLA+ constructs. Note that PGo makes liberal use of temporary variables to compile complex TLA+ constructs.
+
+\begin{center}
+\lstset{style=tabularstyle}
+\renewcommand{\arraystretch}{1.5}
+\begin{longtable}{ || m{0.25\textwidth} | m{0.33\textwidth} | m{0.36\textwidth} || }
+	\hline
+	\textbf{TLA+ feature} & \textbf{Example code} & \textbf{PGo support} \\
+	\hline\hline
+	Function call &
+	\begin{lstlisting}[language=pcal]
+x[exp1]
+\* or
+x[exp1, exp2, exp3]
+\* or
+x[<<exp1, exp2, exp3>>]
+\* or
+x[[field1 |-> e1, field2 |-> e2]]
+	\end{lstlisting} &
+	Supported; compiled code dependent on the type of \lstinline[language=pcal]|x| (the function) \\
+	\hline
+	Binary operator call & \lstinline[language=pcal]|x /\ y = z + 1| & Supported; compiled as expected \\
+	\hline
+	Function (as struct literal) &
+	\begin{lstlisting}[language=pcal]
+[field1 \in exp1 |-> exp2,
+ field2 |-> exp3]
+	\end{lstlisting} &
+	Supported; compiled as sorted slice of key-value structs \\
+	\hline
+	Function set (as function literal) &
+	\begin{lstlisting}[language=pcal]
+[Nat -> Nat]
+\* or
+[Nat -> 1..3]
+	\end{lstlisting} &
+	Unsupported \\
+	\hline
+	Function set (as struct) & \lstinline[language=pcal]|[1..3 -> 1..3]| & Unsupported \\
+	\hline
+	Function substitution &
+	\begin{lstlisting}[language=pcal]
+[f EXCEPT ![exp1] = exp2]
+\* or
+[f EXCEPT !.field = exp]
+	\end{lstlisting} &
+	Unsupported \\
+	\hline
+	If expression &
+	\begin{lstlisting}[language=pcal]
+if condition
+   then thenExp
+   else elseExp
+	\end{lstlisting} &
+	Compiled as
+	\begin{lstlisting}[language=golang]
+var result type;
+if condition {
+	result = thenExp
+} else {
+	result = elseExp
+}
+// result is used in place of
+// the expression hereafter
+	\end{lstlisting} \\
+	\hline
+	Tuple (as slice) & \lstinline[language=pcal]|<<exp1, exp2, exp3>>| & Compiled as slice when all its contents are of the same type
```

Comment:
Do we need to say "as slice" here? The TLA+ feature is just "tuples", from the user perspective.

Reply:
I've added another entry for tuples as structs.